brownie/network/rpc/ganache.py

```diff
@@ -70,7 +70,14 @@ def launch(cmd: str, **kwargs: Dict) -> None:
     print(f"\nLaunching '{' '.join(cmd_list)}'...")
     out = DEVNULL if sys.platform == "win32" else PIPE
 
-    return psutil.Popen(cmd_list, stdin=DEVNULL, stdout=out, stderr=out)
+    try:
+        p = psutil.Popen(cmd_list, stdin=DEVNULL, stdout=out, stderr=out)
+    except FileNotFoundError:
+        print("Error: Failed to start ganache-cli. Check installation!!")
+        print("Insturctions https://github.com/trufflesuite/ganache-cli")
```

Comment:
Please fix this typo "Instructtions -> Instructions"

Reply:
Fixed.